index.md

```diff
@@ -123,8 +123,3 @@ More information regarding accommodation can be found on the [Details page][Deta
 * [SoCraTes France][] / tbc, Rochegude, France
 * [SoCraTes BE][] / tbc, Tournai, Belgium
 
-
-<div style="margin-top:40px;">
-  <a href="/sponsorship.html" class="btn btn-primary btn-lg">Become a sponsor</a>
```

Comment:
why do we want to remove this?

Reply:
It was at the bottom of the page and I don't see how it added any value.
Sponsorship is constantly prominent on the side-bar and the main navigation.

Happy to put it back in if we think it adds value

Reply:
I think it doesn't hurt :) We need more sponsors. But I'm not too fussed.